packages/lab/src/color-chooser/ColorChooser.css

```diff
@@ -11,7 +11,7 @@
 .saltColorChooser-overlayButtonSwatchTransparent {
   height: calc(var(--salt-size-container-spacing) - 2px) !important;
   width: calc(var(--salt-size-container-spacing) - 2px) !important;
-  border: 1px solid var(--salt-status-negative-foreground);
+  border: 1px solid var(--salt-sentiment-negative-foreground-informative);
```

Comment:
This is the wrong token originally but we shouldn't change it in this PR, all of color chooser needs work.

Reply:
if its the same value can it just be kept? so that no deprecated tokens are used throughout